src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -758,9 +763,8 @@ Status MediapipeGraphExecutor::infer(const KFSRequest* requestPtr, KFSResponse*
         }
         outputPollers.emplace(name, std::move(absStatusOrPoller).value());
     }
-
-    std::map<std::string, mediapipe::Packet> inputSidePackets{createInputSidePackets(requestPtr)};
-    absStatus = graph.StartRun(inputSidePackets);
+    std::map<std::string, mediapipe::Packet> sideInputPackets{createInputSidePackets(requestPtr)};
+    absStatus = graph.StartRun(std::move(sideInputPackets));
```

Comment:
Is it really neccessary to move input packets? Mediapipe does not take copy of it, it takes reference